3rdParty/StormLib/src/FileStream.cpp

```diff
@@ -807,6 +810,7 @@ static void BaseHttp_Close(TFileStream * pStream)
 #endif
 }
 
+#ifndef NONET

```

Comment:
NONET is a DevilutionX concept so it shouldn't be used inside of StormLib. I think the right solution would be to make `PLATFORM_WINDOWS && PLATFORM_XBOX` result in the unsupported scenario.